docs/PLUGINS.md

```diff
@@ -262,12 +262,24 @@ If you want to get the configuration combined with your configuration, run `cont
 
 ##### Version header
 
-containerd has two configuration versions:
-- Version 2 (Recommended): Introduced in containerd 1.3.
+containerd has several configuration versions:
+- Version 3 (Recommended for containerd 2.x): Introduced in containerd 2.0.
+  Several plugin IDs have changed in this version.
+- Version 2 (Recommended for containerd 1.x): Introduced in containerd 1.3.
+  Still supported in containerd v2.x.
+  Plugin IDs are changed to have prefixes like "io.containerd.".
 - Version 1 (Default): Introduced in containerd 1.0. Removed in containerd 2.0.
```

Comment:
nod, and if you output the default it's 3 now...

Reply:
I suppose the question is.. what is meant by default here.. default config or default if you have a config that does not specify version =

Reply:
maybe ..Version 1: Introduced in containerd 1.0. Removed in containerd 2.0  Note: if version is not specified in the config the default is version = 1.. otherwise if you do not specify a config or use the config defaults the default for that version of containerd will be provided.

Reply:
Will remove "Default".

The document already says that "A configuration with Version 1 may not have `version` header".

Reply:
fixed